en/docs/learn/configuring-multi-tenancy.md

```diff
@@ -0,0 +1,71 @@
+# Configuring Multi-Tenancy
```

Comment:
Better to give a brief intro to multi-tenancy and point off to a page that discusses it detail.

Reply:
All Field values should be in monospace